### DISCUSSION TITLE
After Latest Update, How Can I Use a $ as my success_symbol or error_symbol?

### DISCUSSION THREAD

**Discussion by csoleimani** [starship/starship#1720] [Q&A] [answered]
It used to work before when it was just "symbol" but now, I get this error when I use a dollar sign

[ERROR] - (starship::config): Unable to parse the config file: invalid escape character in string: `$` at line 6 column 22

~
❯ vim .config/starship.toml

[WARN] - (starship::modules::character): Error in module `character`:
 --> 1:3
  |
1 | [$](bold green)
  |   ^---

How can I escape the dollar sign? Here is my starship.toml

```
# Don't print a new line at the start of the prompt
add_newline = false

# Replace the "❯" symbol in the prompt with "➜"
[character]                            # The name of the module we are configuring is "character"
success_symbol = "[$](bold green)"     # The "success_symbol" segment is being set to "➜" with the color "bold green"

```

**Comment by matchai** [comment, marked as answer]
This has been answered in #1694.

**Reply by csoleimani** [reply]
Sorry I missed that. Thank you!
